src/Components/GenreSelect/GenreSelect.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { buttonStyle, containerStyle, selectedButtonStyle } from "./styles";
```

Comment:
Using `.js` files for styles is not the best approach in the current setup.
Let's discuss it more during our sync on Monday.
Please, check `App.css` file and how it's used.
https://create-react-app.dev/docs/adding-a-stylesheet
You can also use modules to make class names naming simpler (https://create-react-app.dev/docs/adding-a-css-modules-stylesheet).
Or scss - https://create-react-app.dev/docs/adding-a-sass-stylesheet.